test/scripts/readwrite.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <unistd.h>
+
+// To build this just run : gcc readwrite.c -o readwrite
+// To execute this run : ./readwrite
+
+int main()
```

Comment:
Should we run this script in the pipeline or something?

Reply:
No - we shouldn't. This is only for manual testing purposes. We can replicate this with a unit test or something. But I wanted to dump this script here anyway.